### PR TITLE
[notifyjs] Fix `notifyjs is not a module`

### PR DESCRIPTION
**Pull request by NateScarlet**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/alexgibson/notify.js/blob/30583643edea073fbc980a6c91113a3476d4c3a5/README.md>
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/notifyjs/index.d.ts [hunk 1]
```diff
@@ -1,118 +1,111 @@
-// Type definitions for notifyjs 1.2.8
+// Type definitions for notifyjs 3.0.0
 // Project: https://github.com/alexgibson/notify.js
 // Definitions by: soundTricker <https://github.com/soundTricker>
+//                 NateScarlet <https://github.com/NateScarlet>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare var Notify: {
-    new (title : string , options? : notifyjs.INotifyOption): notifyjs.INotify;
+export default class Notify {
```
Comment (andy-ms) [inline types/notifyjs/index.d.ts]: Are you sure this is a default export and not `export = Notify`? See https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Comment (NateScarlet) [reply types/notifyjs/index.d.ts]: https://github.com/alexgibson/notify.js/blob/30583643edea073fbc980a6c91113a3476d4c3a5/src/notify.js#L179
Source code is using `export default`

Comment (NateScarlet) [reply types/notifyjs/index.d.ts]: `default` can be a function or else if i read this doc right: https://www.typescriptlang.org/docs/handbook/modules.html
And i think i am exporting a typescript class which will complie to javascript function at runtime.
Source code is using `export default` and previous version is not importable,  what situation this default property may cause problem?

Comment (andy-ms) [reply types/notifyjs/index.d.ts]: The source code was probably put through a transpiler which generated a commonjs module. But if this is declared as a `default` export, TypeScript code must use a default import, which outputs as `require("notifyjs").default`, which is undefined.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**